sdk/src/types/block/unlock/multi.rs

```diff
@@ -38,6 +38,8 @@ impl MultiUnlock {
     }
 }
 
+impl WorkScore for MultiUnlock {}
```

Comment:
This needs to be the sum of the contained unlocks

Reply:
Fixed